### ISSUE TITLE
[Notebook is running!] Improvements required in code

### ISSUE THREAD

**Issue by iamtalhaasghar** [Prabhutva711/denseonRomanUrduAmazonProb#2]
For anyone who is going to visit this in future. I was able to run this jupyter notebook after applying following changes.

1. `keras.layers.embeddings.Embedding` has been moved to `keras.layers.Embedding` [ref](https://stackoverflow.com/questions/75734680/modulenot-found-keras-layer-embeddings-i-dont-have-any-idea-how-to-fix-this-ev).
2. Rename `yo.csv` to `Roman Urdu DataSet.csv` [dataset file](https://archive.ics.uci.edu/dataset/458/roman+urdu+data+set)
3. Insert / Update column names [`Tweet`, `Feeling`] as dataset doesn't contain them and author has renamed first dataset row to column names which is not the correct way to do it.  The right way is: `df.columns = ['Tweet', 'Feeling']`. Remove the other two `dataset.rename` lines from code.
4. This piece of code is un-necessary. Comment it out as shown:
```
# count=0
# no=0
# for i in dataset['Unnamed: 2'].isnull():
#     if i==True:
#         count=count+1
#     else:
#         no=no+1
# print(count,no)
```
5. In dataset there is a row #13276 which has incorrect label `Neative`. Correct it by using `dataset.loc[13276, 'Feeling'] = 'Negative'`
6. Make sure last layer has only 3 neurons as we have 3 classes / labels. Originally it will be 4 so change it to 3.

Ta da :)
